utils/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
should remove

Reply:
Done in 52ca1ebfed76073302306afa2b1f3d0aa6e80e80